internal/core/software_renderer.rs

```diff
@@ -1398,6 +1409,16 @@ struct GradientCommand {
     bottom_clip: PhysicalLength,
 }
 
+#[derive(Debug)]
+struct ZenoPathCommand {
```

Comment:
(in fact, this is a simplified version of  SharedBufferCommand, for an alphamap)

Reply:
yes. saving as an alpha map reduces the memory size, but makes it heavier on the line drawing function, as gradients need to be calculated there. This shouldn't be a problem as gradient fill on rectangles already does that

Reply:
I meant that the ZenoPathCommand is basically the same as two `SharedBufferCommand { buffer: SharedBufferData::AlphaMap { ... }, ... }`

In fact, the draw_texture_line can be re-used, i think (if you create a `SceneTexture`)